features/staging_commands.feature

```diff
@@ -310,3 +310,68 @@ Feature: Enable command behaviour when attached to an UA staging subscription
            | release |
            | bionic  |
            | xenial  |
+
+    @series.xenial
+    @series.bionic
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Attached enable fips-updates on fips enabled vm
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token_staging` with sudo
+        And I run `ua disable livepatch` with sudo
+        And I run `apt-get install openssh-client openssh-server strongswan -y` with sudo, retrying exit [100]
+        And I run `ua enable fips --assume-yes` with sudo
+        Then stdout matches regexp:
+            """
+            Updating package lists
+            Installing FIPS packages
+            FIPS enabled
+            A reboot is required to complete install
+            """
+        When I run `ua status --all` with sudo
+        Then stdout matches regexp:
+            """
+            fips +yes                enabled
+            """
+        When I reboot the `<release>` machine
+        And  I run `ua enable fips-updates --assume-yes` with sudo
+        Then stdout matches regexp:
+            """
+            Updating package lists
+            Installing FIPS Updates packages
+            FIPS Updates enabled
+            A reboot is required to complete install
+            """
+        When I run `ua status --all` with sudo
+        Then stdout matches regexp:
+            """
+            fips +yes                n/a
+            """
+        And stdout matches regexp:
+            """
+            fips-updates +yes                enabled
```

Comment:
I'm not certain how this test actually passes. on my multipass kvm instance which had fips-updates enabled, I install your latest branch and I see
```bash
SERVICE       ENTITLED  STATUS    DESCRIPTION
esm-infra     yes       enabled   UA Infra: Extended Security Maintenance (ESM)
fips          yes       disabled  NIST-certified FIPS modules
fips-updates  yes       disabled  Uncertified security updates to FIPS modules
livepatch     yes       n/a       Canonical Livepatch service

Enable services with: ua enable <service>

     Account: chad.smith@canonical.com
Subscription: chad.smith@canonical.com
```